src/services/BaseCacheService.ts

```diff
@@ -1,17 +1,43 @@
 import { BaseCacheResource } from '../resources/BaseCacheResource'
-
+const ttl: number = 3_600_000 // 1 hour
```

Comment:
Ideally TTL/Staletime should come in via the constructor of the BaseCacheService